h2o-algos/src/main/java/hex/api/RegisterAlgos.java

```diff
@@ -4,10 +4,13 @@
 import water.H2O;
 import hex.ModelBuilder;
 import water.api.GridSearchHandler;
+import water.rapids.RapidsInit;
 
 public class RegisterAlgos extends water.api.AbstractRegister {
   // Register the algorithms and their builder handlers:
   @Override public void register(String relativeResourcePath) throws ClassNotFoundException {
+    // Register algorithm-specific Rapids
+    RapidsInit.registerAlgoRapids();
```

Comment:
Could we do that via SPI automatically? Or do you have reason to register them explicitly?